main.tf

```diff
@@ -56,6 +56,7 @@ locals {
     "cloudtrace.googleapis.com",
     "container.googleapis.com",
     "servicemanagement.googleapis.com",
```

Comment:
Do we remove this API from the list?

Reply:
Or would doing so be a breaking change?

Reply:
We will leave this API in just in case we need to go back to it again.